tests/devices/default_qubit/test_default_qubit_preprocessing.py

```diff
@@ -788,7 +788,7 @@ def test_u3_non_trainable_params(self):
 
         # U3 decomposes into 5 operators
         assert len(res.operations) == 5
-        assert res.trainable_params == [0, 1, 2, 3, 4]
+        assert res.trainable_params == [0, 1, 2, 3]
```

Comment:
Why this change? It seems that the test is exactly about having all 5 parameters be trainable? :thinking: